dosi-vault-example/src/local-chain-info.js

```diff
@@ -0,0 +1,29 @@
+export const linkCurrency = {
+  coinDenom: "FNSA",
+  coinMinimalDenom: "cony",
+  coinDecimals: 6,
+  coinGeckoId: "link",
+};
+
+export const linkBip44 = { coinType: 438 };
```

Comment:
How about adding the description why the coinType is 438?
https://github.com/satoshilabs/slips/blob/3e7b1032daf6194cde5a243e17ba1f02d8f11f66/slip-0044.md?plain=1#L469